doc/source/user_guide/fields/field_data.rst

```diff
@@ -99,13 +99,13 @@ To retrieve scalar field data, such as absolute pressure, use ``ScalarFieldDataR
 .. code-block:: python
 
   >>> from ansys.fluent.core.services.field_data import ScalarFieldDataRequest
-  >>> scalar_request_1 = ScalarFieldDataRequest(field_name="absolute-pressure", surfaces=["inlet"])
-  >>> abs_press_data = field_data.get_field_data(scalar_request_1)
+  >>> absolute_pressure_request = ScalarFieldDataRequest(field_name="absolute-pressure", surfaces=["inlet"])
+  >>> absolute_pressure_data = field_data.get_field(absolute_pressure_request)
```

Comment:
@prmukherj `get_field` is a tricky name here -- it creates an expectation that the return type is some kind of array.

Reply:
Will update it to "get_field_data" again.

Reply:
Thanks